cmd/erasure-server-pool.go

```diff
@@ -299,7 +301,7 @@ func (p serverPoolsAvailableSpace) FilterMaxUsed(max int) {
 
 	// Remove entries that are above.
 	for i, z := range p {
-		if z.MaxUsedPct < max {
+		if z.Valid && z.MaxUsedPct < max {
```

Comment:
```suggestion
		if z.MaxUsedPct > 0 && z.MaxUsedPct < max {
```

Reply:
@harshavardhana It seems to me this would break on new installs. Maybe @vadmeste can explain the original code.

Reply:
yeah it is risky for me to ignore pools with 0% of usage, such as (0.11%)